src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -1,6 +1,11 @@
 import os
 import robot
 from robot.errors import DataError
+try:
+    from robot.utils import DotDict
+except ImportError:  # New in RF 2.9
+    DotDict = dict
```

Comment:
While this works, I agree with @pekkaklarck. We should just do our own implementation of DotDict, even though it'll be a bit more work. That will remove the 2.9 dependency which could be potentially confusing to users who don't read the documentation well.